dev/TabView/TestUI/TabViewPage.xaml.cs

```diff
@@ -56,6 +57,13 @@ public TabViewPage()
             DataBindingTabView.TabItemsSource = itemSource;
         }
 
+        protected async override void OnNavigatedTo(Windows.UI.Xaml.Navigation.NavigationEventArgs args) 
+        {
+            NotCloseableTab.Visibility = Visibility.Collapsed;
+            await Task.Delay(1);
```

Comment:
Please use an override that takes a TimeSpan to make this more readable (so I don't have to go to the docs to see what this does 🙂).

```suggestion
            await Task.Delay(TimeSpan.FromMilliseconds(1));
```